src/h5web/providers/hsds/api.ts

```diff
@@ -180,12 +168,6 @@ export class HsdsApi implements ProviderAPI {
       attributes,
       children,
     };
-
-    children.forEach((c) => {
-      c.parent = group;
-    });
```

Comment:
We no longer need to find the root group from any entity, so we no longer have to keep parent references.